dll/keyboard/CMakeLists.txt

```
@@ -60,6 +60,7 @@ list(APPEND _keyboard_layouts
     kbdru
     kbdru1
     kbdsg
+    kbdsf
```

```suggestion
    kbdsf
    kbdsg
```
(classify by lexicographical order)